src/modules/flight_mode_manager/tasks/AutoFollowTarget/follow_target_params.c

```diff
@@ -44,15 +44,15 @@
  */
 
 /**
- * Minimum follow target altitude
+ * Follow target altitude
  *
- * The minimum height in meters relative to home for following a target
+ * The height in meters for following a target
  *
  * @unit m
  * @min 8.0
  * @group Follow target
  */
-PARAM_DEFINE_FLOAT(NAV_MIN_FT_HT, 8.0f);
+PARAM_DEFINE_FLOAT(NAV_FT_HT, 8.0f);
```

Comment:
So far I avoided renaming existing follow-me parameters. If we are going to rename them, we might as well completely redefine them. For example, we could remove the `NAV_` since it's no longer part of navigator.

If we rename existing parameters, we'll also have to update mavsdk to reflect the renaming.

Reply:
Gotcha, and yes we can remove NAV. And maybe do : FOLLOW_TARGET_*

Reply:
`FOLLOW_TARGET_` is way too long, yes :sweat_smile: 

You could swap `NAV_FT` with `FLW_TGT`, or `FLW_ME`.

By the way, I still haven't made up my mind whether we should call it follow-target or follow-me :roll_eyes: 
Follow-me sounds much friendlier.

Reply:
FLW_TGT sounds nice 👍